extension/packages/nextjs/app/eip-5792/_components/EIp5792Example.tsx

```diff
@@ -0,0 +1,157 @@
+"use client";
+
+import { useState } from "react";
+import { EIP5972TxNotification } from "./EIP5792TxNotification";
+import { useAccount, useConnect } from "wagmi";
+import { useCapabilities, useWriteContracts } from "wagmi/experimental";
+import { InputBase } from "~~/components/scaffold-eth";
+import { useDeployedContractInfo, useScaffoldWriteContract } from "~~/hooks/scaffold-eth";
+import { getParsedError, notification } from "~~/utils/scaffold-eth";
+
+export const EIP5792Example = () => {
+  // wagmi hooks not EIP-5792 specific
+  const { address, isConnected, chainId } = useAccount();
+  const { connectAsync, connectors } = useConnect();
+
+  const [newGreetings, setNewGreetings] = useState("");
+
+  // Scaffold-ETH hooks to get contract abi, address and write to contract (not EIP-5792 specific)
+  const { data: YourContract } = useDeployedContractInfo("EIP5792_Example");
+  const { writeContractAsync: writeYourContractAsync, isPending: isWriteYourContractPending } =
+    useScaffoldWriteContract("EIP5792_Example");
+
+  // wagmi hook to batch write to multiple contracts (EIP-5792 specific)
+  const { writeContractsAsync, isPending: isBatchContractInteractionPending } = useWriteContracts();
+
+  const isLoading = isWriteYourContractPending || isBatchContractInteractionPending;
+
+  //  isSuccess is true if the wallet is EIP-5792 compliant
+  const { isSuccess: isEIP5792Wallet, data: walletCapabilites } = useCapabilities({ account: address });
+
+  const handleBatchTransaction = async () => {
+    try {
+      if (!YourContract || !walletCapabilites || !chainId) return;
+
+      if (newGreetings.trim().length === 0) {
+        notification.error("Please input new greetings");
+        return;
+      }
+
+      const isPaymasterSupported = walletCapabilites?.[chainId]?.paymasterService?.supported;
+      // (OPTIONAL) An ERC-7677 paymaster service URL if you want to sponsor gas fee
+      const paymasterURL = "";
+
+      const txnId = await writeContractsAsync({
+        contracts: [
+          {
+            address: YourContract.address,
+            abi: YourContract.abi,
+            functionName: "setGreeting",
+            args: [newGreetings],
+          },
+          {
+            address: YourContract.address,
+            abi: YourContract.abi,
+            functionName: "increaseCounter",
+          },
+        ],
+        capabilities:
+          isPaymasterSupported && Boolean(paymasterURL)
+            ? {
+                paymasterService: {
+                  url: `${paymasterURL}/api/paymaster`,
```

Comment:
why to add `/api/paymaster` here?
I tried to use paymaster URL from coinbase docs https://portal.cdp.coinbase.com/products/bundler-and-paymaster. It works by default, but doesn't work with `/api/paymaster`. Default URL is `https://api.developer.coinbase.com/rpc/v1/base-sepolia/<YOUR ID>`